src/Symfony/Component/Yaml/Inline.php

```diff
@@ -604,10 +604,19 @@ private static function evaluateScalar($scalar, $flags, $references = array())
                         return self::evaluateBinaryScalar(substr($scalar, 9));
                     case preg_match('/^(-|\+)?[0-9,]+(\.[0-9]+)?$/', $scalar):
                         return (float) str_replace(',', '', $scalar);
-                    case preg_match(self::getTimestampRegex(), $scalar):
+                    case preg_match(self::getTimestampRegex(), $scalar, $datetimeMatches):
                         if (Yaml::PARSE_DATETIME & $flags) {
-                            $date = new \DateTime($scalar);
```

Comment:
Given @nicolas-grekas comment, it could simply be
 ```php
$date = new \DateTime($scalar, new \DateTimeZone('UTC'));
$date->setTimeZone(new \DateTimeZone('UTC'));

return $date;
```